CONTRIBUTING.md

```diff
@@ -1,20 +1,20 @@
-## How to contribute to Drucker project
+## How to contribute to Rekcurd project
 
-First of all, thank you so much for taking your time to contribute! Drucker is not very different from any other open
+First of all, thank you so much for taking your time to contribute! Rekcurd is not very different from any other open
 source projects you are aware of. It will be amazing if you could help us by doing any of the following:
 
-- File an issue in [the issue tracker](https://github.com/drucker/drucker/issues) to report bugs and propose new features and
+- File an issue in [the issue tracker](https://github.com/rekcurd/python-rekcurd/issues) to report bugs and propose new features and
```

Comment:
I would prefer `rekcurd-python`. gRPC uses similar naming, i.e. `grpc-go` etc

Reply:
OK! Thank you for your quick review!